### ISSUE TITLE
[Feature Request] Server-side waypoint sharing.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [rfresh2/XaeroPlus#141]
https://github.com/rfresh2/XaeroPlus/blob/c37503e0b312f95fa00c93df22ac36fce4acb122/README.md#modifications doesn't list anything relevant. There is desire for this, per https://www.reddit.com/r/feedthebeast/comments/s8q6l7/is_there_a_option_too_see_other_players_waypoints/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button.

**Comment by rfresh2** [comment]
There is no automatic waypoints sharing feature in XaeroPlus or the base mods.

There is a way to share waypoints in chat which other players can click on to add but that's it.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/rfresh2/XaeroPlus/issues/141#issuecomment-2183523969

@rfresh2, indeed, hence the request, since this repository appears to exist to improve it. The absence of any ability so display Waypoints between players is a reason that I and my friends are being forced to search for alternatives, but I much prefer this map mod to all the competitors I've used thus far...

**Comment by rfresh2** [comment]
Ok, wasn't clear if this was a feature request or a support question.

XaeroPlus is currently a client-side only mod but its something i've considered before.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/rfresh2/XaeroPlus/issues/141#issuecomment-2183552158

@rfresh2, I've also filed this at the original mod's issue tracker - https://legacy.curseforge.com/minecraft/mc-mods/xaeros-world-map/issues/859 - so that if it's accepted there, maintenance of this feature shan't be your burden unnecessarily.

**Comment by rfresh2** [comment]
yeah its probably for the best if xaero does it

there's actually some code already in the base mods referring to `serverWaypoints` but its not fully implemented.

its been there since at least late 2022 when i started tracking changes but i guess he just hasn't got around to finishing it
